ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CoverageEmptyState.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Text, Bullseye, Flex, FlexItem, PageSection } from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
+
+import EmptyStateTemplate from 'Components/EmptyStateTemplate';
+import { complianceEnhancedSchedulesPath } from 'routePaths';
+
+import CoveragesPageHeader from './CoveragesPageHeader';
+
+function CoverageEmptyState() {
+    return (
+        <>
+            <CoveragesPageHeader />
+            <PageSection isFilled>
+                <Bullseye className="pf-v5-u-background-color-100">
+                    <EmptyStateTemplate
+                        title="No scan data available"
+                        headingLevel="h2"
+                        icon={CubesIcon}
+                    >
+                        <Flex direction={{ default: 'column' }}>
+                            <FlexItem>
+                                <Text>
+                                    Create a scan schedule to assess profile compliance on selected
+                                    clusters.
+                                </Text>
+                            </FlexItem>
```

Comment:
What do you think about `hasWriteAccessForCompliance` as conditional rendering for this sentence?

The link itself is okay, even with Read access only.

Reply:
I think it's a good idea. Updated.